public/styles/common.css

```diff
@@ -204,51 +204,63 @@ div#content {
 /********************/
 
 button {
-    color: #333;
-    background: white;
-    border: 1px solid #CCC;
-    border-radius: 4px;
-    padding: 0.3em 0.6em;
-    display: inline-block;
-    text-decoration: none;
-    cursor: pointer;
+	color: #333;
+	background: white;
+	border: 1px solid #CCC;
+	border-radius: 4px;
+	padding: 0.3em 0.6em;
+	display: inline-block;
+	text-decoration: none;
+	cursor: pointer;
+}
+
+button.primary {
+	color: white;
+	font-weight: bold;
+	background: #B03931;
+	border-color: #98312A;
+}
+
+button.primary:hover, button.primary:focus {
+	background: #742620;
+	border-color: #943029;
 }
 
 /*********************/
 /* PACKAGE CLIPBOARD */
 /*********************/
 
 .clipboard-pkg {
-    padding-top: 5px;
+	padding-top: 5px;
 }
 
 .clipboard-pkg-title {
-    font-weight: bold;
-    display: inline-block;
-    width: 60px;
+	font-weight: bold;
+	display: inline-block;
+	width: 60px;
 }
 
 .clipboard-pkg-row input {
-    width: 280px;
+	width: 280px;
 }
 
 /********************/
 /* COPY SHIELDS     */
 /********************/
 
 #copy-shields div {
-    padding-left: 10px;
+	padding-left: 10px;
 }
 
 #copy-shields .img-wrapper {
-    width: 140px;
-    display: inline-block;
-    padding-left: 5px;
-    vertical-align: middle;
+	width: 140px;
+	display: inline-block;
+	padding-left: 5px;
+	vertical-align: middle;
 }
 
 #copy-shields input {
-    width: 280px;
+	width: 280px;
```

Comment:
Please do unrelated changes in a separate commit.

Reply:
hm there were like only 2 occurences of this where they were spaces instead of tabs and they were right near my changes so I didn't want to work with 2 different levels of indentation

Reply:
like they are just very few changes and they will never get changed otherwise

Reply:
I personally don't mind, it's just a policy at the main D repositories, but I agree that traffic is pretty low here.